src/main/java/org/broadinstitute/hellbender/tools/spark/sv/ReadClassifier.java

```diff
@@ -113,8 +113,9 @@ private void checkDiscordantPair( final GATKRead read, final List<BreakpointEvid
             evidenceList.add(new BreakpointEvidence.InterContigPair(read, readMetadata));
         } else if ( read.isReverseStrand() == read.mateIsReverseStrand() ) {
             evidenceList.add(new BreakpointEvidence.SameStrandPair(read, readMetadata));
-        } else if ( (read.getStart() < read.getMateStart() && read.isReverseStrand()) ||
-                    (read.getStart() > read.getMateStart() && !read.isReverseStrand()) ) {
+        } else if ( (read.getEnd() < read.getMateStart() && read.isReverseStrand()) ||
+                    (read.getStart() > read.getMateStart() + read.getLength() && !read.isReverseStrand()) ) {
```

Comment:
@tedsharpe  I'd been loath to do that (add some sort of short allowable overhang between pairs) since I didn't feel like adding yet another hard-coded magic number. But, after further consideration, I think you're right, and setting it to 10 seems to get close to, but not quite to, the same effect on the number of intervals called, so we're likely still filtering out the bad stuff but maybe keeping a little good stuff that we weren't before. I've committed the change and will merge when tests pass.